src/node_contextify.cc

```diff
@@ -185,6 +185,30 @@ Local<Context> ContextifyContext::CreateV8Context(
   CHECK(name->IsString());
   Utf8Value name_val(env->isolate(), name);
 
+  Local<Value> codegen = options_obj->Get(env->context(),
+      FIXED_ONE_BYTE_STRING(env->isolate(), "codeGeneration"))
+          .ToLocalChecked();
+
+  if (!codegen->IsUndefined()) {
+    CHECK(codegen->IsObject());
```

Comment:
We could leave a comment here, that the js part already ensured that we have `undefined` or an object. Looking only at the C++ part, my first response was that this `CHECK` would fail a lot.

Reply:
this pattern it used a lot with stuff that is validated js-side and we don't leave usually comments, are you worried about something specific not being construed?